lib/utils.go

```diff
@@ -0,0 +1,21 @@
+package lib
+
+import (
+	"os"
+)
+
+// FileExists checks if a file exists and is not a directory before we try using it to prevent further errors.
```

Comment:
nitpick for consistency
```suggestion
// FileExists checks if a file exists and is not a directory before we try using it to prevent further errors
```